spec/classes/init_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'spec_helper'
+describe 'Sentry' do
+  let(:facts) do
```

Comment:
The puppetlabs-apache module splits on, at least, the `operatingsystemrelease` fact:
```
Puppet::PreformattedError:
       Evaluation Error: Error while evaluating a Function Call, 'split' parameter 'str' expects a String value, got Undef at .../modules/sentry/spec/fixtures/modules/apache/manifests/version.pp:7:16
```
And we require the custom `python_version` fact exposed from stankevich-python in our `sentry::wsgi` class:
```
     Puppet::PreformattedError:
       Evaluation Error: Error while evaluating a Function Call, 'regsubst' parameter 'target' expects a value of type Array or String, got Undef at .../modules/sentry/spec/fixtures/modules/sentry/manifests/wsgi.pp:59:13
```
The included facts are more comprehensive than what's actually needed, though. I'll pare it down.  It looks like all we need are:
```
osfamily: 'RedHat',
operatingsystem: 'RedHat',
operatingsystemrelease: '7.2',
python_version: '2.7.5'
```